training/src/pages/Trainee/Trainee.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import AddDialog from './components/AddDialog/AddDialog';
+
+export default class Trainee extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+    };
+  }
+
+  handleClickOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = () => {
+    const { open } = this.state;
+    this.setState({ open: false });
+    return open;
+  };
+
+  handleSubmit = (data) => {
```

Comment:
on submit the Dialog does not closed.
also handleSubmit is not working properly.

Reply:
resolved